integration-tests/smallrye-reactive-messaging/src/main/java/org/apache/camel/quarkus/component/smallrye/reactive/messaging/it/SmallRyeReactiveMessagingResource.java

```diff
@@ -52,6 +54,8 @@ public Response post(String message) throws Exception {
     @Produces(MediaType.TEXT_PLAIN)
     @GET
     public String getValues() {
-        return String.join(",", results.getResults());
+        List<String> values = results.getResults();
+        Collections.sort(values);
```

Comment:
Is this the actual fix?

Reply:
Yes. The order in which the messages are processed is not important. We just need to verify that all of the expected values are there.